keys/ed25519.go

```diff
@@ -0,0 +1,64 @@
+package keys
```

Comment:
would it make sense for this to be under `crypto`? I'm unsure what the difference is

Reply:
yeah it could be, that probably makes more sense. I will move it